Examples/CMakeLists.txt

```diff
@@ -20,6 +20,11 @@ set(CMAKE_CXX_STANDARD 11)
 # popen()/pclose() are not C++ standards
 set(CMAKE_CXX_EXTENSIONS ON)
 
+if(PCAPPP_BUILD_COVERAGE AND NOT ANDROID)
+  add_compile_options(--coverage)
```

Comment:
You don't need to add it here no? Except if you build examples out of three

Reply:
It raised link errors in some CI runs while working on it. I'll check again, might the behavior of the ci/compile has changed due to other PRs.

Reply:
Yes we are building tutorials out of three here, so as libpcap++ has been built with coverage and not linked.

We could either build libraries as shared libraries or test the building of tutorial out-of three in another jobs ?

Reply:
My bet is on shared libraries because I aimed to gather coverage of all ci workflows individually without changing them. I mean, what we test what we miss when a PR run. So, separating tutorials to another job might lead to misinterpretation and also increase the CI work.